types/src/front/assistant/actions/websearch.ts

```diff
@@ -13,27 +13,32 @@ export type WebsearchConfigurationType = {
   description: string | null;
 };
 
-export const WebsearchResultSchema = t.type({
+const WebsearchRunOutputResultSchema = t.type({
   title: t.string,
   snippet: t.string,
-  link: t.string,
+  sourceUrl: t.string,
```

Comment:
Does this even work with legacy websearch entries?

Reply:
I'm a bit antsy about changing types just to please models. We can surely render differently at the moment we present to the model instead no?

Reply:
Naming it `sourceUrl` is consistent with all other data sources so I thought it was cleaner as is, so it's not only changing types just to please models.

I can definitely roll back the dust app to put back "link" and then tweak only where we need if you think it's better.